### ISSUE TITLE
Frame size (1195725856) larger than max length (67108864) in query

### ISSUE THREAD

**Issue by qiaojialin** [apache/iotdb#2664]
v0.11.2

This is caused by a group by time query or a raw data query that contains one time series:

```
2021-02-04 09:23:23,330 [pool-7-IoTDB-RPC-Client-thread-499] ERROR org.apache.thrift.server.TThreadPoolServer$WorkerProcess:321 - Thrift Error occurred during processing of message.
org.apache.thrift.transport.TTransportException: Frame size (1195725856) larger than max length (67108864)!
        at org.apache.thrift.transport.TFastFramedTransport.readFrame(TFastFramedTransport.java:152)
        at org.apache.thrift.transport.TFastFramedTransport.read(TFastFramedTransport.java:136)
        at org.apache.thrift.transport.TTransport.readAll(TTransport.java:86)
        at org.apache.thrift.protocol.TBinaryProtocol.readAll(TBinaryProtocol.java:455)
        at org.apache.thrift.protocol.TBinaryProtocol.readI32(TBinaryProtocol.java:354)
        at org.apache.thrift.protocol.TBinaryProtocol.readMessageBegin(TBinaryProtocol.java:243)
        at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:27)
        at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:313)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
2021-02-04 10:12:30,788 [pool-7-IoTDB-RPC-Client-thread-497] ERROR org.apache.thrift.server.TThreadPoolServer$WorkerProcess:321 - Thrift Error occurred during processing of message.
org.apache.thrift.transport.TTransportException: java.net.SocketException: Connection reset
        at org.apache.thrift.transport.TIOStreamTransport.read(TIOStreamTransport.java:127)
        at org.apache.thrift.transport.TTransport.readAll(TTransport.java:86)
        at org.apache.thrift.transport.AutoExpandingBufferReadTransport.fill(AutoExpandingBufferReadTransport.java:37)
        at org.apache.thrift.transport.TFastFramedTransport.readFrame(TFastFramedTransport.java:156)
        at org.apache.thrift.transport.TFastFramedTransport.read(TFastFramedTransport.java:136)
        at org.apache.thrift.transport.TTra
```

**Comment by antzhb** [comment]
i have the same problem with v1.3

**Comment by myysophia** [comment]
I have the same problem with v1.2.2. The error is very frequent, but does not affect reading and writing


![image](https://github.com/apache/iotdb/assets/25994521/c9f7a5a7-1043-4d71-a434-3323d7d777fb)

**Comment by qiaojialin** [comment]
This is fixed in 1.3.1: https://iotdb.apache.org/Download/

**Comment by JackieTien97** [comment]
Ite seems that it's caused by the huge query result whose single frame size is larger than the default client max frame size(default is 64MB).
You can change that field `Session.thriftMaxFrameSize`
